updater/updater.php

```diff
@@ -146,19 +167,18 @@
 			$loader->assign("BACKUP_STEP_THREE", xlt("Restoring DB"));
 			$loader->assign("EXIT_UPDATER", xlt("Return To Updater"));
 			$files_need_to_be_downloaded = array();
-			foreach ($merged_requests_array as $key => $value) {
-				$pr_number = $value;
+		
+				$pr_number = $cpr_number;
 				$arr = getSinglePullRequestFileChanges($updater_token, $repository_owner, $repository_name,  $pr_number);
 				foreach ($arr as $ke) {
 					$sha = $ke['sha'];
 					$time = time();
 					$extension = pathinfo($ke['filename'], PATHINFO_EXTENSION);	
 					array_push($files_need_to_be_downloaded, $sha);	
 				}
-			}
 			$count_files = count($files_need_to_be_downloaded);
 			$loader->assign("STATUS", xlt("status"));
-			$loader->assign("UPDATER_START", "<b>$cpr_title</b><br/><h6>$cpr_body</h6>");
+			$loader->assign("UPDATER_START", "<b>$cpr_title(#$cpr_number)</b><br/><h6>$cpr_body</h6>");
 			$loader->assign("COUNT_FILES", $count_files);
 			$loader->assign("PR_NUMBER", xlt("Write Feedback For #".$pull_request_number));
```

Comment:
Same here.  translate tool will not work with mixed variable data

Reply:
working on it @tmccormi  :relaxed: